schema.sql

```diff
@@ -0,0 +1,147 @@
+                                    rating INT(63),
+                                    user_role_id INT,
+                                    city_id INT,
+                                    user_status_id INT
+);
+
+CREATE TABLE IF NOT EXISTS user_role (
+                                         id INT AUTO_INCREMENT PRIMARY KEY,
+                                         title VARCHAR(79) NOT NULL UNIQUE
+);
+
+CREATE TABLE IF NOT EXISTS city (
+                                    id INT AUTO_INCREMENT PRIMARY KEY,
+                                    title VARCHAR(192) NOT NULL UNIQUE
+);
+
+CREATE TABLE IF NOT EXISTS user_status (
+                                           id INT AUTO_INCREMENT PRIMARY KEY,
+                                           title VARCHAR(79) NOT NULL UNIQUE
+);
+
+CREATE TABLE IF NOT EXISTS reviews (
+                                       id INT AUTO_INCREMENT PRIMARY KEY,
+                                       description VARCHAR(255) NOT NULL,
+                                       creation_date DATETIME NOT NULL DEFAULT CURRENT_TIMESTAMP,
+                                       task_id INT,
+                                       grade_id INT
+);
+
+CREATE TABLE IF NOT EXISTS grade (
+                                     id INT AUTO_INCREMENT PRIMARY KEY,
+                                     value INT(8) NOT NULL UNIQUE
+);
+
+CREATE TABLE IF NOT EXISTS task (
+                                    id INT AUTO_INCREMENT PRIMARY KEY,
+                                    title VARCHAR(79) NOT NULL,
+                                    description VARCHAR(159) NOT NULL,
+                                    budget INT,
+                                    creation_date DATETIME NOT NULL DEFAULT CURRENT_TIMESTAMP,
+                                    expiration_date DATETIME,
+                                    address_description VARCHAR(255),
+                                    task_status_id INT,
+                                    category_id INT,
+                                    customer_id INT,
+                                    worker_id INT,
+                                    task_address_id INT
+);
+
+CREATE TABLE IF NOT EXISTS task_address (
+                                            id INT AUTO_INCREMENT PRIMARY KEY,
+                                            latitude FLOAT(15),
+                                            longitude FLOAT(15),
+                                            city_id INT
+);
+
+CREATE TABLE IF NOT EXISTS task_status (
+                                           id INT AUTO_INCREMENT PRIMARY KEY,
+                                           title VARCHAR(79) NOT NULL
+);
+
+CREATE TABLE IF NOT EXISTS task_files (
+                                          id INT AUTO_INCREMENT PRIMARY KEY,
+                                          file VARCHAR(255) NOT NULL,
+                                          task_id INT
```

Comment:
NOT NULL
\ + type INT NOT NULL DEFAULT 1 (is_image TINYINT(1) NOT NULL DEFAULT 1)